doc/lua_api.txt

```diff
@@ -6844,9 +6844,16 @@ object you are working with still exists.
         * `"plain"`: Uses 0 textures, `bgcolor` used
     * `clouds`: Boolean for whether clouds appear in front of `"skybox"` or
       `"plain"` custom skyboxes (default: `true`)
-* `get_sky()`: returns base_color, type, table of textures, clouds.
-* `get_sky_color()`: returns a table with the `sky_color` parameters as in
-    `set_sky`.
+* `get_sky(as_table):
+    * returns a table containing sky parameters as defined in
+    `set_sky(sky_parameters)`.
+    * `as_table`: boolean to avoid calling the deprecated version.
```

Comment:
Nowhere does this say that you'd have to set the boolean to true...
Might be better to merge the two function entries to clearly explain which usage is deprecated and which isn't.